docs/tutorials/setting-up-an-nfs-file-share.md

```diff
@@ -97,56 +97,54 @@ After creating the file, the `nfsd` daemon [should automatically start up][OSXSe
 
 If you make changes to `/etc/exports`, activate them with `nfsd update`.
 
-Now update the fstab on the EV3.  
-
 ## How To Do It - EV3
 
-On the EV3 we first need to enable and start NFS modules. Type these commands on the command line:
-
-    sudo systemctl enable nfs-common.service
-    sudo systemctl start nfs-common.service
-    sudo systemctl enable rpcbind.service
-    sudo systemctl start rpcbind.service
-
-Next you'll need to update a file (as root) called `/etc/fstab`. You should have already set up USB Networking, so `ssh` to the EV3 and run an editor like `vi` or `nano` to edit the file. Here's the line you want to add to `/etc/fstab` - DO NOT TOUCH ANYTHING ELSE IN THERE!
-
+On the client - ev3dev - side, we need to create a system mount unit in order to mount our newly created NFS share. **Note**: The classic way of mounting the NFS share via an entry in /etc/fstab does not work!
```

Comment:
"system mount unit"

Could be more clear if you say "systemd `.mount` file". I suspect that the term "unit file" is probably not familiar to most people.

Reply:
Good idea to have a note about `fstab`. We can make it stand out even more with some special formatting.

```
{% include /style/icon.html type="warning" %}
The classic way of mounting the NFS share via an entry in /etc/fstab does not work!
{: .alert .alert-warning}
```